src/resolver/mod.rs

```diff
@@ -31,12 +33,12 @@ pub type Result<T> = ::std::result::Result<T, Error>;
 ///
 /// The `Resolver` side is a client of the `Executor`. Namerd work is performed on
 /// whatever thread the executor is spawned on.
-pub fn new(namerd: Namerd) -> (Resolver, Executor) {
+pub fn new<N: Namer + 'static + Send>(namer: N) -> (Resolver, Executor) {
```

Comment:
IMHO, using `where` clauses for long bounds like this can be more readable.
```rust
pub fn new<N>(namer: N) -> (Resolver, Executor) 
where N: Namer + Send,
      N: 'static,

```

Reply:
what's the guidance on how to group these or break them up?  I notice you have Namer + Send together.

Reply:
I decided to group the trait bounds separately from the lifetime bound, but
```rust
where N: Namer,
      N: Send,
      N: 'static,
```
would also be acceptable – I don't think there's a strongly codified guideline here.